kategory-docs/docs/docs/datatypes/try/README.md

```diff
@@ -4,6 +4,171 @@ title: Try
 permalink: /docs/datatypes/try/
 ---
 
-## Try 
+## Try
+
+Kategory has a lots of different types of error handling and reporting, which can make it difficult to decide which one is best for your situation. 
+
+For example, we have `Option` to model the absence of a value, or `Either` to model the return of a function as a type that may have been successful, or may have failed. 
+
+On the other hand, we have `Try`, which represents a computation that can result in an `A` result (as long as the computation is successful) or in an exception if something has gone wrong. 
+
+That is, there are only two possible implementations of `Try`: a `Try` instance where the operation has been successful, which is represented as `Success<A>`; or a `Try` instance where the computation has failed with a `Throwable`, which is represented as `Failure<A>`.
+
+With just this explanation you might think that we are talking about an `Either<Throwable, A>`, and you are not wrong. `Try` can be implemented in terms of `Either`, but its use cases are very different.
+
+If we know that an operation could result in a failure, for example, because it is code from a library over which we have no control, or better yet, some method from the language itself. We can use `Try` as a substitute for the well-known `try-catch`, allowing us to rise to all its goodness.
+
+The following example represents the typical case when consuming Java code, where domain errors are represented with exceptions.  
+
+```kotlin:ank
+open class GeneralException: Exception()
+
+class NoConnectionException: GeneralException()
+
+class AuthorizationException: GeneralException()
+
+data class User(val id: Long, val name: String)
+
+fun checkPermissions() {
+    throw AuthorizationException()
+}
+
+fun getLotteryNumbersFromCloud(): List<User> {
+    throw NoConnectionException()
+}
+
+fun getLotteryNumbers: List<User> {
+    checkPermissions()
+    
+    return getLotteryNumbersFromCloud()
+}
+```
+
+The traditional way to control this would be to use a `try-catch` block, as we have said before:
+
+```kotlin:ank
+try {
+    getLotteryNumbers()
+} catch (e: NoConnectionException) {
+    println("You don't have connection")
+} catch (e: AuthorizationException) {
+    println("You don't have permission")
+}
+```
+
+However, we could use `Try` to retrieve the computation result in a much cleaner way:
+
+```kotlin:ank
+import kategory.*
+
+val lotteryTry = Try { getLotteryNumbers() }
+lotteryTry
+```
+
+By using `getOrElse` we can give a default value to return, when the computation fails, similar to what we can also do with `Option` when there is no value:
+
+```kotlin:ank
+lotteryTry.getOrElse { emptyList() }
+```
+
+We can also use `recover` which allow us to recover from a particular error (we receive the error and have to return a new value):
+
+```kotlin:ank
+lotteryTry.recover {
+    if (it is NoConnectionException) emptyList()
+    else throw it
```

Comment:
this is not a good example because it promotes exception throwing. We should instead here return another `Try` or use `recoverWith`

Reply:
or coalesce to a List regardless of the exception type

Reply:
https://github.com/kategory/kategory/pull/458/commits/fdb5790ea2a0658a4de1f55c187f6dc18098566e